### ISSUE TITLE
Not respecting minOccurs="0" of group element

### ISSUE THREAD

**Issue by ccwang002** [brunato/xmlschema#34]
Starting 0.9.15+, XML from https://www.uniprot.org/uniprot/P50222.xml and its schema from https://www.uniprot.org/docs/uniprot.xsd failed the validation due to zero occurrence of group element in a complex type element. 

```python
>>> import xmlschema
>>> schema = xmlschema.XMLSchema('https://www.uniprot.org/docs/uniprot.xsd')
>>> schema.validate('./P50222.xml')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/liang/miniconda3/envs/ptm_preprocess/lib/python3.6/site-packages/xmlschema/validators/xsdbase.py", line 367, in validate
    raise error
xmlschema.validators.exceptions.XMLSchemaChildrenValidationError: failed validating <Element '{http://uniprot.org/uniprot}interactant' at 0x11bf21868> with <XsdGroup '{http://uniprot.org/uniprot}interactantGroup' at 0x11be02630>.

Reason: The content of element 'interactant' is not complete. Tag 'id' expected.

Schema:

    <xs:group xmlns:xs="http://www.w3.org/2001/XMLSchema" minOccurs="0" ref="interactantGroup" />

Instance:

    <interactant xmlns="http://uniprot.org/uniprot" intactId="EBI-748397" />

```

The relevant definitions are 

```xml
<xs:element name="interactant" type="interactantType" minOccurs="2" maxOccurs="2"/>
<xs:group name="interactantGroup">
    <xs:sequence>
        <xs:element name="id" type="xs:string"/>
        <xs:element name="label" type="xs:string" minOccurs="0"/>
    </xs:sequence>
</xs:group>
<xs:complexType name="interactantType">
    <xs:group ref="interactantGroup" minOccurs="0"/>
    <xs:attribute name="intactId" type="xs:string" use="required"/>
</xs:complexType>
```

I expect an `interactant` does not require any occurrence of `interactantGroup`. The same XML can be validated without any errors in the previous releases. Sorry I still learning the XML schema and was unable to pin point the cause in the code base.  Please let me know what you think. 

Thanks for making this tool. It is extremely useful.

**Comment by brunato** [comment]
Found the bug, it was in the `XsdGroup.is_emptiable()` method that didn't check the `min_occurs` value.
I'll insert the fix into next release (0.9.18).

Thanks for your appreciations. I knew it wouln't be easy to build a new library for XML Schema but the challenge seems feasible now.

**Comment by brunato** [comment]
The version 0.9.18 that resolves this issue is available on PyPI.
I close the issue.
Best regards
